packages/webpack/test/fixtures/main/main.js

```diff
@@ -24,4 +29,4 @@ function run() {
   )
 }
 
-run()
+run(x)
```

Comment:
why is this diff importnat?

Reply:
I'm preventing the URL from being unused and therefore skipped.